esrally/rally.py

```diff
@@ -611,17 +618,34 @@ def print_help_on_errors():
 
 
 def race(cfg):
-    other_rally_processes = process.find_all_other_rally_processes()
-    if other_rally_processes:
-        pids = [p.pid for p in other_rally_processes]
-        msg = "There are other Rally processes running on this machine (PIDs: %s) but only one Rally benchmark is allowed to run at " \
-              "the same time. Please check and terminate these processes and retry again." % pids
-        raise exceptions.RallyError(msg)
+    logger = logging.getLogger(__name__)
+
+    kill_running_processes = cfg.opts("system", "kill.running.processes", default_value=False, mandatory=False)
+
+    if kill_running_processes:
+        logger.info("Killing running Rally processes")
+
+        # Kill any lingering Rally processes before attempting to continue - the actor system needs to be a singleton on this machine
+        # noinspection PyBroadException
+        try:
+            process.kill_running_rally_instances()
+        except BaseException:
+            logger.exception(
+                "Could not terminate potentially running Rally instances correctly. Attempting to go on anyway.")
+    else:
+        other_rally_processes = process.find_all_other_rally_processes()
+        if other_rally_processes:
+            pids = [p.pid for p in other_rally_processes]
+
+            msg = "There are other Rally processes running on this machine (PIDs: %s) but only one Rally benchmark " \
```

Comment:
We are using `.format` and recently [f-strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) so can you please rewrite this as:

```python
msg = f"There are other Rally processes running on this machine (PIDs: {pids}) but only one Rally " \
      f"benchmark is allowed to run at the same time.\n\nPlease rerun with --kill-running-processes to " \
      f"terminate them automatically."
```

(I also suggested a slightly different wording)